lib/OpenQA/Scheduler.pm

```diff
@@ -14,20 +14,13 @@
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
 package OpenQA::Scheduler;
+use Mojo::Base 'Mojolicious';
 
-use strict;
-use warnings;
```

Comment:
Is that now implicit with `Mojo::Base 'Mojolicious'` or not?

Reply:
Yes, this is a [very important feature](https://mojolicious.org/perldoc/Mojo/Base#DESCRIPTION) of `Mojo::Base`.

Reply:
Perl::Critic is aware of this as well btw.